app/mailers/user_mailer.rb

```diff
@@ -4,6 +4,19 @@ class UserMailer < ApplicationMailer
   SIGN_IN_EMAIL_TEMPLATE = "7ab8db5b-9842-4bc3-8dbb-f590a3198d9e"
   EMAIL_CONFIRMATION_TEMPLATE = "50059d26-c65d-4e88-831a-8bfb9f4116cd"
   PRIMARY_CONTACT_TEMPLATE = "a7cc4d19-c0cb-4187-a71b-1b1ea029924f"
+  TUTOR_NOMINATION_TEMPLATE = "240c5685-5cb0-40a9-9bd4-1a595d991cbc"
```

Comment:
This has already been done. Check the school mailer it already has the mailer you need to call

Reply:
Done